ChainSafe/Multix
# Balance conversion for balance types

closes #375 

This adds the token at the end of the field for manual extrinsics in case the field is expecting a balance. It will then convert the value in plank with the right amount of 0s.

To test:
- Craft a manual extrinsic that needs a value, for instance `balances.transfer` or `bounties.proposeBounty`
- check that the token is visible
- check that when the tx is done, the right amount in plank is submitted. Note that for balances.transfer, the overview converts it back to human readable values, like in my screenshot

Submitting
![image](https://github.com/ChainSafe/Multix/assets/33178835/63d87f57-ae20-4668-a2ba-e79bc160b208)

Verifying
![image](https://github.com/ChainSafe/Multix/assets/33178835/7b5e8b84-2570-4244-ac1b-7700bb4795da)
